moveit_ros/hybrid_planning/global_planner/global_planner_plugins/src/moveit_planning_pipeline.cpp

```diff
@@ -74,6 +74,7 @@ bool MoveItPlanningPipeline::initialize(const rclcpp::Node::SharedPtr& node)
   node->declare_parameter<double>(PLAN_REQUEST_PARAM_NS + "max_velocity_scaling_factor", 1.0);
   node->declare_parameter<double>(PLAN_REQUEST_PARAM_NS + "max_acceleration_scaling_factor", 1.0);
   node->declare_parameter<std::string>("ompl.planning_plugin", "ompl_interface/OMPLPlanner");
+  node->declare_parameter<std::string>("ompl.request_adapters", "UNDEFINED");
```

Comment:
I guess this should actually be like this to match the convention for the other params below
```suggestion
  node->declare_parameter<std::string>("ompl.request_adapters", UNDEFINED);
```